inputs/org.osbuild.containers

```diff
@@ -63,6 +80,16 @@ SCHEMA = r"""
       }
     }
   },
+  "source-storage-ref": {
+    "type": "object",
+    "additionalProperties": false,
+    "minProperties": 1,
+    "patternProperties": {
+      ".*": {
+        "$ref": "#/definitions/source-storage-options"
+      }
+    }
+  },
```

Comment:
This is a problem.  I think that because both `source-object-ref` and `source-storage-ref` have the same pattern property, it's not clear which one will "win" the `oneOf` in L157, so it tries to match an object-ref with the storage-ref schema and fails.

See this:
```
› sudo osbuild -l . test/data/stages/skopeo/a.json
test/data/stages/skopeo/a.json has errors:

.pipelines[2].stages[1].inputs.images:
  {'type': 'org.osbuild.containers', 'origin': 'org.osbuild.source', 'references': {'sha256:dbb63178dc9157068107961f11397df3fb62c02fa64f697d571bf84aad71cb99': {'name': 'manifest-list-test'}}} is not valid under any of the given schemas
```

If, just for the sake of testing, we change the pattern property here
```
› git diff
diff --git a/inputs/org.osbuild.containers b/inputs/org.osbuild.containers
index 83c0fbb7..f8390774 100755
--- a/inputs/org.osbuild.containers
+++ b/inputs/org.osbuild.containers
@@ -85,7 +85,7 @@ SCHEMA = r"""
     "additionalProperties": false,
     "minProperties": 1,
     "patternProperties": {
-      ".*": {
+      "whatever.*": {
         "$ref": "#/definitions/source-storage-options"
       }
     }
```
then it works
```
› sudo osbuild -l . test/data/stages/skopeo/a.json
build:          dc6e3a66fef3ebe7c815eb24d348215b9e5e2ed0cd808c15ebbe85fc73181a86
input-tree:     e5b19c6aa8ea8fb1a065c3261a45917e98f76a2599cf9d8d9a5b8c8ea6e093af
tree:           d26da71cee02a4d3389270fc7063845b6b3eb287b04d03035b660a533f213cfb
```